core/arch/x86/decode_table.c

```diff
@@ -4011,6 +4475,10 @@ const instr_info_t prefix_extensions[][8] = {
     {OP_vmovdqu, 0xf30f6f10, "vmovdqu", Vx, xx, Wx, xx, xx, mrm|vex, x, tpe[113][5]},
     {OP_vmovdqa, 0x660f6f10, "vmovdqa", Vx, xx, Wx, xx, xx, mrm|vex, x, tpe[113][6]},
     {INVALID,   0xf20f6f10, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID, 0x0f6f10, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID, 0xf306f10, "vmovdqu32/64", xx, xx, xx, xx, xx, no, x, NA}, /* W0/W1 */
```

Comment:
u32, u64 are 2 different opcodes in DR. There a distinct based on VEX_W (see Intel Specs)

Reply:
You are right. They have different opcodes. They differs only in EVEX.W bit. I wrote in this style only for hint about that  entry. That will be changed.

Reply:
@hgreving2304 please resolve discussion if I have answered your question.